core/app/models/spree/tax/output/item.rb

```diff
@@ -0,0 +1,10 @@
+module Spree
+  module Tax
+    module Output
+      class Item
+        include ActiveModel::Model
+        attr_accessor :id, :tax_rate, :amount, :included_in_price
```

Comment:
1. I would like the `label` to be part of this. Avalara can add labels to whatever they return, and so should we.
2. I would like the matching not to be done by ID. When recording taxation API interactions with e.g. Avalara, it always hits us that matching is done by what is essentially a database artifact. Can we just try relying on sorting by ID and adding taxes - or no taxes - based on that? 
3. I would like our tax rate to be a hidden implementation detail. If using an external service, e.g. Avalara, we don't have tax rates to play with. 
```
attr_accessor :i_of_sorted_items, :label, :amount, :included_in_price
```